src/utils/nviHelpers.ts

```diff
@@ -8,3 +17,125 @@ export const getNviYearFilterValues = () => {
   }
   return nviYearFilterValues;
 };
+
+const areTopLevelOrgsEqual = <T>(set1: Set<T>, set2: Set<T>) => {
```

Comment:
rename denne også kanskje?
```suggestion
const isEqualSets = <T>(set1: Set<T>, set2: Set<T>) => {
```